### DISCUSSION TITLE
Docstring format

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#9] [Polls]
Which format do you prefer?
1. [Google style](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)
2. [NumPy style](https://numpydoc.readthedocs.io/en/latest/format.html)
3. Others?

**Comment by BECATRUE** [comment]
I'm used to Google style rather than NumPy style.

**Reply by kangz12345** [reply]
Me too!
